cli/commands/team23.go

```diff
@@ -0,0 +1,51 @@
+/*
+Root command is created as team23. Holds no functionality but all other
+commands are built on top of this command. Creation of new commands 
+requires an init function per command with rootCmd.AddCommand(<newCmd>)
+*/
+
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"github.com/spf13/cobra"
+	"github.com/19chonm/461_1_23/cli/functionality"
+)
+
+
+var rootCmd = &cobra.Command{
+	Use: "team23",
+	Short: "team23 - root command for app",
+	Long: "team23 is the root command to navigate through Team 23's CLI",
+	Run: func(cmd *cobra.Command, args[]string){
+
+	},
+}
+
+
+// test URL_FILE with: /Users/emile/461_1_23/test/urls_file.txt
+
+// First function to be ran on main. Will check if second argument is either
+// an absolute filepath, one of the recognized commands or neither. If neither,
+// program will throw error. If argument is an absolute filepath, a direct call
+// to functions are executed. No cobra command is created because name varies.
+func Execute() {
+	
+	if filepath.IsAbs(os.Args[1]) {
```

Comment:
If I run `./run`, I get the following error:

```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/19chonm/461_1_23/cli/commands.Execute()
        /home/shay/a/schwa123/ECE461/cli/commands/team23.go:36 +0x14b
main.main()
        /home/shay/a/schwa123/ECE461/main.go:6 +0x17
```

Looking at the error line, my guess is the fact that we are are using `os.Args[1]` without checking to see if it exists first. Could we add in a check to see if it exists first? If it doesn't, we should have it error on line 46 where you have written "Not a recognized command"

Reply:
Agreed.